src/components/FolderPath/FolderPath.tsx

```diff
@@ -4,7 +4,7 @@ import { ChevronRightIcon } from '@/icons/icons';
 import { IconSize } from '@/types/enums';
 
 interface Props {
-  path: (React.JSX.Element | string)[];
+  path: React.ReactNode[];
```

Comment:
This one to me makes more sense conceptually if it is Jsx elements and string, the idea being that you can either have a name for the folder path or an icon